src/main/java/com/academy/task/config/EmailConfig.java

```diff
@@ -0,0 +1,36 @@
+package com.academy.task.config;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.mail.javamail.JavaMailSender;
+import org.springframework.mail.javamail.JavaMailSenderImpl;
+import java.util.Properties;
+
+@Configuration
+public class EmailConfig {
+
+    @Value("${spring.mail.username}")
+    private String email;
+
+    @Value("${spring.mail.password}")
+    private String password;
+
+    @Bean
+    public JavaMailSender getJavaMailSender() {
```

Comment:
`spring-boot-starter-mail` will preconfigure this component automatically. all you have to do is to add relevant configuration in `application.properties`